src/composables/latestTransactionList.ts

```diff
@@ -28,14 +40,39 @@ export function useLatestTransactionList({ store }: IDefaultComposableOptions) {
 
   const tokens = computed(() => store.state.fungibleTokens.tokens);
 
+  const localPendingTransactionsList = computed(
+    (): IPendingTransaction[] => {
+      const { networkId } = store.getters.activeNetwork;
+      return (store.state.transactions as ITransactionsState).pending[networkId] || [];
+    },
+  );
+
+  function buildLatestTransaction() {
+    const transactions = [...fetchedTransactions.value];
+    pendingTransactions.value.forEach((transaction) => {
+      if (!transactions.some(({ hash }) => transaction.hash === hash)) {
+        transactions.unshift(transaction as ITransaction);
```

Comment:
`transaction` has different type and forcing TS to change it is in my opinion a bad thing. If we want to allow to put pending transactions into the collection of transactions we need to change how the `transactions` are typed.